src/notifier.py

```diff
@@ -19,12 +19,11 @@
 
 from logger import Logger
 
-logger = Logger("notifier")
-
 
 class cmd_run(Command):
     help = "Listen for events and report them on stdout"
     args = [Args.db_config]
+    _logger = Logger("config/logger.conf", "notifier")
```

Comment:
Oh I was thinking of creating one in the `__init__` constructor rather than making it a class attribute.  So if later on we want to have command line arguments to specify things like the logger config file path or the logging levels then we could use that when constructing the object.

Reply:
I see you're doing this in `runner.py` actually - so we should just add an `__init__()` in this class too and do the same.  In fact we may even make a derived class from `Command` with the logger support, or a mixin.  Then the classes in the pipeline scripts would inherit from it to avoid duplicating the initialisation etc. everywhere.

Reply:
I added constructors to `notifier`, `send_kcidb`, and `trigger` scripts.